lib/octicons_gem/lib/octicons_v2/octicon_v2.rb

```diff
@@ -84,5 +81,25 @@ def calculate_width(height)
     def calculate_height(width)
       (width.to_i * @height) / @width
     end
+
+    def get_octicon(symbol, options)
```

Comment:
There's probably a newer cooler style of doing this, but you should guard against `options` build `nil` so line 87 doesn't fail when it is

```suggestion
    def get_octicon(symbol, options = {})
```